pontoon/administration/templates/admin_project.html

```diff
@@ -53,9 +53,8 @@ <h3>Locales</h3>
   <div class="locales clearfix">
       {{ admin_team_selector.render(locales_available, locales_selected, locales_readonly) }}
       {{ form.locales }}
-      {{ form.locales.errors }}
+      {{ form.non_field_errors() }}
       {{ form.locales_readonly }}
-      {{ form.locales_readonly.errors }}
```

Comment:
I think we should keep `{{ form.locales.errors }}` and `{{ form.locales_readonly.errors }}` in case there are any field-specific errors.